tests/phpunit/test-wc-stripe-sub-renewal.php

```diff
@@ -64,6 +64,8 @@ public function test_renewal_successful() {
 		$payments_intents_api_endpoint = 'https://api.stripe.com/v1/payment_intents';
 		$urls_used                     = array();
 
+		$renewal_order->set_payment_method('stripe');
```

Comment:
Hey, this is likely a dumb question, but I'm not sure I get why this is needed?

Reply:
https://github.com/woocommerce/woocommerce-gateway-stripe/blob/master/includes/abstracts/abstract-wc-stripe-payment-gateway.php#L350

If not manually set, the order defaults to a different payment gateway, cheque payments IIRC.